docs: add semantic PR, conventional commit info, link to .github docs

drecali opened the pull request:
**- What I did**
I added a mention about using conventional commits in CONTRIBUTING.md, and included a link to the relevant website (`https://www.conventionalcommits.org`). I did this because I saw that #50 did not use conventional commits and one of the maintainers [suggested](https://github.com/atsign-foundation/at_demos/pull/50#pullrequestreview-1084786159) it's best to use conventional commits in the future.

**- How I did it**
I added an instruction about conventional commits and the website `https://www.conventionalcommits.org`

**- How to verify it**
Check the Markdown preview and click to link to confirm it works.
If I missed something or need to change the wording, please let me know. I wasn't sure if it's an absolute requirement, so I phrased it more loosely. If it is a strict requirement, maybe it's better to use something like `Contributors must follow the conventional commits standard`.

**- Description for the changelog**
Add conventional commits info to CONTRIBUTING.md

drecali committed:
chore: add conventional commit info

drecali committed:
chore: simplify conventional commits URL

drecali commented:
@TylerTrott I saw in a few recent commits to other `atsign` repos that contributors were requested to use semantic/conventional commits/PRs. It's mentioned in a few comments such as this one:
> One resource looking into is [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) 
> ([Source](https://github.com/atsign-foundation/at_app/pull/85#issuecomment-1044382607))

I didn't see it mentioned in the contributing instructions so that's why I added it. If it's in the contributing instructions, more contributors will probably use it, which will save more maintainer time 🤞 

Since the end of Hacktoberfest is approaching, is it possible for a maintainer to review this PR? If any changes are required, I'm happy to oblige.

Thanks!

cpswan commented:
Hi @drecali 

We have more documentation of how we use GitHub, including the bot and Conventional Commits at https://github.com/atsign-foundation/.github/blob/trunk/atGitHub.md but you're right that we should have more about it (even if it's just a link to that page) in the CONTRIBUTING.md

The way we'd normally approach this is changing the template repo, then using a script to update all the repos generated from the template. But I can take care of transporting any changes here back into the template and onwards from there.

drecali commented:
Sorry about that @cpswan! I didn't realize that's how you manage content in `CONTRIBUTING.md`. Should I close this PR or are improvements required? Are there next steps or other contribution opportunities I should be aware of?

cpswan commented:
There's nothing to be sorry about @drecali, we welcome your contribution.

Can you please update this PR by adding a further link to https://github.com/atsign-foundation/.github/blob/trunk/atGitHub.md explaining that this provides further background on how we use GitHub.

drecali committed:
docs: add semantic PR mention, link to .github repo

Link to atsign-foundation GitHub organization's conventions and configurations.

drecali commented:
Thanks @cpswan! I noticed the link you shared already included a link to the Conventional Commits site so I replaced my link with yours. I also included a link to the `.github` repo mentioning conventions and configs. Please let me know if more changes are required.

cpswan commented:
Thanks @drecali
LGTM
Approved and merged :)
I'll use this as the basis for an updated template, and across the existing live repos.

drecali commented:
Thanks @cpswan! You made it easy to contribute :)
Just wondering, will you have a chance to review #52 soon as well? It's not large but I think it can be helpful for beginners.
